langfuse-core/src/index.ts

```diff
@@ -98,7 +99,8 @@ abstract class LangfuseCoreStateless {
   abstract setPersistedProperty<T>(key: LangfusePersistedProperty, value: T | null): void;
 
   constructor(params: { publicKey: string; secretKey?: string } & LangfuseCoreOptions) {
-    const { publicKey, secretKey, ...options } = params;
+    const { ...options } = params;
+    const { publicKey, secretKey } = utils.configLangfuseSDK(params);
```

Comment:
I get an error here:
```
Property 'configLangfuseSDK' does not exist on type 'typeof utils_d'.ts
```

Reply:
We have so far been unable to reproduce this error, can you please share some more context on this?